discord/message.py

```diff
@@ -1706,13 +1832,22 @@ def __init__(
 
         self.interaction: Optional[MessageInteraction] = None
 
+        # deprecated
```

Comment:
I can do it myself but it might be worth adding `@discord.utils.deprecated` to the older attribute by changing it into a read-only property to apply the decorator.

Reply:
Done!